README.md

```diff
@@ -88,6 +88,7 @@
 - [vspect](https://github.com/zakuro9715/vspect) - A tool to inspect vlang source file. ( Archived )
 - [vzcc](https://github.com/malisipi/vzcc) - A CLI cross-compiling tool based on Zig CC for V.
 - [vin](https://github.com/DeoDorqnt387/vin) - A Basic Command Line Interface for V.
+- [vfetch](https://github.com/carlosqsilva/vfetch) - A macOS system information fetch written in V.
```

Comment:
Please put entries in alphabetical order.  Yes, I know `vin` is out of place, but that can be fixed.

Reply:
fixed